### PR TITLE
feat: adding key status "expired" handler

### PR DESCRIPTION
**Pull request by mr-pinzhang**
### Checklist

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] API or design changes are documented in API.md

### CODE REVIEW

File: src/controller/eme-controller.js [hunk 1]
```diff
@@ -4,6 +4,8 @@
  * DRM support for Hls.js
  */
 
+/* eslint-disable indent */
```
Comment (toxic-johann) [inline src/controller/eme-controller.js:7]: Why do we need this?

File: src/controller/eme-controller.js [hunk 2]
```diff
@@ -337,14 +339,28 @@ class EMEController extends EventHandler {
...
+            details: ErrorDetails.KEY_SYSTEM_LICENSE_EXPIRED,
+            fatal: true
+          });
+          break;
+        }
+        case 'output-downscaled':
+        case 'output-restricted': {
+          logger.error(`Key session (status: ${status}) is not in a valid status.`);
+
+          this.hls.trigger(Event.ERROR, {
```
Comment (toxic-johann) [inline src/controller/eme-controller.js:357]: Do we also need to log the key status?

**Review by toxic-johann — COMMENTED** [review]
Is this PR finished? Looks like we only add the event to log information here.

File: src/controller/eme-controller.js (hunk 1 above)
Comment (mr-pinzhang) [reply src/controller/eme-controller.js:7]: related to
https://github.com/Tubitv/hls.js/pull/12/files#diff-1cb8b36ed42a658ab63ac970ab1bd648d67f9b199f95fd279f8a50c8d51df2a6R343-R357

I'm not sure why it'll report `indent` error for this `switch` block.

File: src/controller/eme-controller.js (hunk 2 above)
Comment (mr-pinzhang) [reply src/controller/eme-controller.js:357]: I don't think it's necessary in this case.

**Review by toxic-johann — APPROVED** [review]

**Pull request merged**